sentences/nl/homeassistant_HassTurnOn.yaml

```diff
@@ -3,24 +3,29 @@ intents:
   HassTurnOn:
     data:
       - sentences:
-          - "[<doe>|<zou>] <name> [<naar>] aan [willen|kunnen] [<doe>]"
-          - "schakel <name> [<naar>] in"
+          - "[<doe>|<zou>] <name>[ ][<type>] [<naar>] (aan|in) [willen|kunnen] [<doe>]"
+          - "schakel <name>[ ][<type>] [<naar>] in"
+          - "[<zou>] <name>[ ][<type>] [willen|kunnen] (aan[ ]zetten|inschakelen)"
+
       - sentences:
           - "open <name>"
           - "[<doe>|<zou>] <name> <open> [willen|kunnen] [<doe>]"
         response: "cover"
+
       - sentences:
           - "open <name> [in|op] <area>"
           - "[<doe> |<zou> ]<name> (<open> [in|op] <area>|[in|op] <area> <open>) [willen|kunnen] [<doe>]"
           - "<zou> <name> [in|op] <area> (willen|kunnen) <open>"
         response: "cover_area"
+
       - sentences:
```

Comment:
this one should be moved to the cover file?

Reply:
already done in your own PR